x-pack/plugins/ess_security/public/common/components/landing_cards/index.tsx

```diff
@@ -73,13 +84,25 @@ export const LandingCards = memo(() => {
   );
 
   const href = useMemo(() => prepend(addIntegrationsUrl), [prepend, addIntegrationsUrl]);
+
   return (
-    <EuiFlexGroup data-test-subj="siem-landing-page" direction="column" gutterSize="l">
-      <EuiFlexItem>
+    <EuiFlexGroup
```

Comment:
The page padding for the "Get Started" page only needs to be added here in the ESS version. It can be done using a `EuiPageTemplate.Section` for example:

```suggestion
    <EuiPageTemplate.Section paddingSize="l">
      <EuiFlexGroup
        [...]
```